website/content/docs/enterprise/federation.mdx

```diff
@@ -31,3 +31,11 @@ can make queries to the remote datacenter in service of both API and DNS
 requests for remote resources (in spite of the partially-connected nature of the
 topology as a whole). Consul datacenters can simultaneously participate in both
 network areas and the existing WAN pool, which eases migration.
+
+## Requirements
+
+- Consul Enterprise 0.8+
+- [Direct connectivity between all Consul servers](https://www.consul.io/docs/k8s/installation/multi-cluster#traditional-wan-federation)
+  - Verify that [`connect.enable_mesh_gateway_wan_federation`](/docs/agent/options#connect_enable_mesh_gateway_wan_federation)
+    is set to `false` (default) to ensure cross-datacenter control plane traffic is direct, not
+    [funneled through mesh gateways](https://www.consul.io/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways)
```

Comment:
If the bulleted-list format is too limiting for the nuance you want to describe, then I would do something like this:

## Requirements

The following conditions are required to enable WAN federated for Consul Enterprise deployments.

### Version

Consul Enterprise 0.8+ is required.

### Control Plan Requirements

Consul servers must have direct connections to each other (see <LINK>). Control plane traffic must have direct routes and cannot be funneled through mesh gateways. Verify that connect.enable_mesh_gateway_wan_federation is set to false (default).

---
It might be confusing to link to a topic that we're telling users NOT to implement, i.e., the mesh gateway topic and the hub-and-spoke tutorial referenced earlier.

Reply:
> It might be confusing to link to a topic that we're telling users NOT to implement, i.e., the mesh gateway topic and the hub-and-spoke tutorial referenced earlier.

Is the suggestion that I remove the link but otherwise keep the text?

> If the bulleted-list format is too limiting for the nuance you want to describe

Is the suggestion that I use this alternate format instead of sub-bullets (if I feel a sub-bullet is needed)? I ask because the conversation on Slack on this topic suggested that this alternate format was too verbose for the content being conveyed, and that the short bulleted list was probably a better choice.

Reply:
Yes, the consensus in Slack was to use bullets. The problem with having these discussions in that platform is that we lose a lot of the context that inform our decisions. So, yes, I think if you need the room to describe additional context, then I would go ahead an use the 'long-form' requirements. 

Also, yes, I think we should remove the link to information we're telling users not to implement.

Reply:
> Also, yes, I think we should remove the link to information we're telling users not to implement.

I think I included that link in case someone wanted to learn more about what that meant (or didn't even know it was an option in Consul). I wasn't expecting most users to click on this if it's not something they intend to implement, but I wanted them to be able to learn more about it easily if they wanted. Should I still remove that link?

Reply:
There isn't _a lot_ of additional context required. It's the same as on the other page: one of the bullets warrants a second sentence of elaboration (given in a sub-bullet).

I agree that we'd want the long-form version if there was a lot of context required. But in this case, I'm leaning towards leaving it as a concise, bulleted list:

![image](https://user-images.githubusercontent.com/85913323/155226889-53758f60-cc62-46bb-82ea-fefc15802612.png)